main/app.py

```diff
@@ -0,0 +1,336 @@
+from __future__ import print_function
+import random
+import sqlite3
+import sys
+import tkFileDialog
+from Tkinter import Tk
+
+from clint.textui import colored, puts
+from colorama import init, Back, Style  # Fore
+from pyfiglet import figlet_format
+from termcolor import cprint
+
+from .models.room import Office, LivingSpace
+from .util.File import FileParser
+import ipdb
+
+
+class Amity(object):
+    """Description for amity"""
+
+    def __init__(self):
+        # variable used in create_rooms
+        self.rooms = {
+            'O': [],
+            'L': []
+        }
+        # variable used in add_person
+        self.people_data = {'Staff': [],
+                            'Fellow': []
+                            }
+        self.allocate = {
+            'O': {' ': []},
+            'L': {' ': []}
+        }
+
+        self.room_type = "O" or "L"
+        self.name = ""
+        name = self.name
+        self.personnel_type = ""
+        personnel_type = self.personnel_type
```

Comment:
**Code Quality: Nitpick**

I thought we agreed on one-line multiple assignments for this.